Projects/Animals.md

```diff
@@ -46,8 +46,8 @@ A foal taking its first pilates class.
 
 Mama Bear and Baby Bear cuddle.
 
-## Cats
+## Flamingo 
 
-![Cat musicians](https://media.giphy.com/media/t7MWRoExDRF72/giphy.gif)
+![Baby Flamingo GIF](https://media.giphy.com/media/ygbGMpSxSkyo8/giphy.gif)
```

Comment:
The funniest part about this is that you are overwriting @lfrustaci's work, in a way not so dissimilar to the ways of the Mama flamingo.

Reply:
How do I pull the request?